bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyEventServlet.java

```diff
@@ -49,7 +48,6 @@
  *
  * @author Markus Michels - Initial contribution
  */
-@NonNullByDefault
```

Comment:
This is unexpected. Why remove the annotation ? Should be better to handle the warnings, if any.